fluentui_listitem/src/main/java/com/microsoft/fluentui/tokenized/tabItem/TabItem.kt

```diff
@@ -118,9 +119,15 @@ fun TabItem(
     val iconContent: @Composable () -> Unit = {
         Icon(
             imageVector = icon,
-            modifier = Modifier.size(if (textAlignment == TabTextAlignment.NO_TEXT) 28.dp else 24.dp),
+            modifier = Modifier.size(if (textAlignment == TabTextAlignment.NO_TEXT) 28.dp else 24.dp)
+                        .graphicsLayer(alpha = 0.99f)
+                        .drawWithCache {
+                            onDrawWithContent {
+                                drawContent()
+                                drawRect(brush = iconColorBrush, blendMode = BlendMode.SrcAtop)
```

Comment:
how is this blendMode value chosen?

Reply:
We overlay a gradient rectangle on top of the Icon Vector to allow gradient colors, that's why we use BlendMode.SrcAtop
You can read more here: https://developer.android.com/reference/kotlin/androidx/compose/ui/graphics/BlendMode#SrcAtop()